src/utils/xmlfileUnits.cpp

```diff
@@ -402,6 +402,9 @@ unsigned long XMLfileUnits::getNodeValueUnit(const char* nodepath, XMLfileUnits:
 	{
 		value=XMLfileUnits::ValueUnit(v,u);
 	}
+	else {
+		value = v;
+	}
```

Comment:
can you please add a comment what the purpose of this if / else is?

Reply:
You mean in a comment incode like this?
```suggestion
	else { /* The XML tag did not include a unit attribute, so use the unscaled value. */
		value = v;
	}
```